less/v2/npc.less

```diff
@@ -5,8 +5,11 @@
 .dnd5e2.sheet.actor.npc {
   --dnd5e-sheet-header-height: 68px;
   --dnd5e-sheet-sidebar-width: 180px;
-  // min-width: 700px;
-  // min-height: 700px;
+  min-width: 700px;
+  min-height: 700px;
```

Comment:
I had these commented out because they break minimization. I attempted an approach of only having them when the sheet wasn't minimized, but that caused the minimization animation to break. Is there a proper way to handle minimum sheet sizes in AppV2 (we didn't set a minimum size for Compendium Browser too for this reason)?

Reply:
There should be a way to do it: I remember going through some effort to allow App V2 journal sheet to have a min-width but also be minimisable. I'll have a look later.

Reply:
`&:not(.minimized, .minimizing, .maximizing)` ?

Reply:
Probably worth noting that AppV2 journal sheet does *not* have the above and it does not animate minimizing/maximizing.

https://github.com/foundryvtt/foundryvtt/issues/12316

Reply:
Zhell's suggestion works